libcudacxx/.upstream-tests/test/std/atomics/atomics.lockfree/isalwayslockfree.pass.cpp

```diff
@@ -97,7 +99,7 @@ void run()
     CHECK_ALWAYS_LOCK_FREE(void*);
     CHECK_ALWAYS_LOCK_FREE(float);
     CHECK_ALWAYS_LOCK_FREE(double);
-    CHECK_ALWAYS_LOCK_FREE(long double);
+    //CHECK_ALWAYS_LOCK_FREE(long double);
```

Comment:
Is this a leftover?

Reply:
No, for whatever reasons, the suppression of the warning does not work in this single test. I tried all kinds of position of the suppression but decided that it does not add any value and simply disabled the offending line

Reply:
Should it be disable for any particular CTK version?

Reply:
Yes we generally do not support long double and it is ifdefed out in most tests

Reply:
Could you leave a comment that long doubles are not supported before the commented line?

Reply:
Added a comment